### ISSUE TITLE
Flow Mapping Mechanic

### ISSUE THREAD

**Issue by 39Nova** [ppy/osu#11797]
*Peppy suggested I make a feature RQ so this idea doesn't get forgotten about*

Idea of a “flow” mechanic in the view tab where you could see the general path of your map stylised with arrows.
I believe this could be beneficial for mappers as an alternate view to see awkward turns/jumps.

E.g.: here, spaces represent 1/4th of a beat. , a sudden gap in arrows indicates a faster jump. This could also be highlighted with a change in colour, fading to red to signify an increase in speed/stress.
![image](https://user-images.githubusercontent.com/79141196/108050083-12786f80-7041-11eb-9fb4-dd07e01617be.png)

You could also highlight small yellow circles and yellow lines to signify the sliders and hit-circles, or alternatively just have them at lower opacity behind or in-front of this overlay.

**Comment by bdach** [comment]
I have zero clue what the picture is showing. What are those axes even, hit circle positions?

**Comment by 39Nova** [comment]
I already showed it to peppy so I just gave the brief description, lemme explain:

arrow signifies 1/4th of the beat, distance of arrows signifies the distance between hit circles. Arrows/line points follow the map the way the cursor would flow, like the follow points in gameplay do. For example in the image above would be 10 hit circles going diagonally up the screen, the number of arrows could also change depending on your current mapping setting, e.g, 1/16th setting would put an arrow at every 1/16th of the beat.
